src/main.ts

```diff
@@ -215,7 +220,7 @@ export default class RecipeGrabber extends Plugin {
 				// magicTime datePublished
 			} else if (arguments.length == 3) {
 				// magicTime datePublished "dd-mm-yyyy HH:MM"
-				if (new Date(arg1) == "Invalid Date") {
```

Comment:
this check won't ever return true. So, added a `isValidDate` function above, so it can be shared.

Reply:
Wasn't sure how you wanted to fix the reorder, do you want to pull the branch, or you can just make the suggested change, and i can merge.